clo-workflows/Scripts/src/service/dataService/SpDataService.ts

```diff
@@ -102,9 +102,10 @@ export class SpDataService implements IDataService {
     async fetchClientActiveProjects(client: User): Promise<Array<CloRequestElement>> {
         const activeProjects: Array<CloRequestElement> = await this.getHostWeb()
             .lists.getByTitle(ListName.PROJECTS)
-            .items.get(this.cloRequestElementParser)
+            .items.filter(this.ACTIVE_FILTER_STRING)
+            .get(this.cloRequestElementParser)
 
-        return activeProjects.filter(item => item.submitter === client.name)
+        return activeProjects.filter(item => item.submitterId === client.name)
```

Comment:
I don't understand this filter, it seems like the submitterId would never equal the user.name